Dockerfile

```diff
@@ -1,26 +1,28 @@
-FROM mono:3.12.1 as builder
+FROM mono:6.8
 ARG CHOCOVERSION=stable
 
-RUN echo "deb http://archive.debian.org/debian/ wheezy main contrib non-free" >/etc/apt/sources.list
+# Make sure we have these tools
 RUN apt-get update && apt-get install -y wget tar gzip
 
+# Download, extract, and move chocolatey installer
 WORKDIR /usr/local/src
-RUN wget "https://github.com/chocolatey/choco/archive/${CHOCOVERSION}.tar.gz"
-RUN tar -xzf "${CHOCOVERSION}.tar.gz"
-RUN mv "choco-${CHOCOVERSION}" choco
+RUN wget "https://github.com/chocolatey/choco/archive/${CHOCOVERSION}.tar.gz" && \
+    tar -xzf "${CHOCOVERSION}.tar.gz" && \
+    rm "${CHOCOVERSION}.tar.gz" && \
+    mv "choco-${CHOCOVERSION}" choco
 
+# Build chocolatey
 WORKDIR /usr/local/src/choco
 RUN chmod +x build.sh zip.sh
 RUN ./build.sh -v
 
+# Symlink the build output to our install directory
+RUN ln -s /usr/local/src/choco/code_drop/chocolatey /opt/chocolatey
 
-FROM debian:stretch
-RUN apt-get update && apt-get install -y mono-devel curl && apt-get clean all
-RUN curl https://ssl-ccp.godaddy.com/repository/gdig2.crt.pem -o gdig2.crt.pem
-RUN cat gdig2.crt.pem >> /etc/ssl/certs/ca-certificates.crt
-RUN cert-sync /etc/ssl/certs/ca-certificates.crt
-COPY --from=builder /usr/local/src/choco/build_output/chocolatey /opt/chocolatey
+# Copy in the choco helper script
 COPY bin/choco /usr/bin/choco
 
+ENV ChocolateyInstall /opt/chocolatey
```

Comment:
Can you point me in the direction of the documentation for this value? I'm assuming you've tested the behavior such that it emulates my trap in the bash script below?

Reply:
I have tested it, and it does work. 

There is currently not documentation for it, since there is no documentation for linux installs. I do have an issue open for it along with the other steps for install- 
https://github.com/chocolatey/choco/issues/2025

I figured it out by looking at the windows install process, and seeing that it set the `chocolateyinstall` env variable on windows. There is documentation for this, under installing to a different location- https://chocolatey.org/courses/installation/installing?method=installing-to-a-different-location